### PR TITLE
[FEATURE] create header

### PR DESCRIPTION
**Pull request by fshovchko**
Added header component with logout action. Component is responsible and changes if screen is less than 1024 pixels wide. Also replaced .png logo with .svg

### CODE REVIEW

File: src/pages/profile/profile.jsx [hunk 1]
```diff
@@ -20,6 +21,7 @@ const Profile = () => {
 
   return (
     <div className="routingButtons">
+      <Header />
```
Comment (mtalstykh-exadel) [inline src/pages/profile/profile.jsx]: Add the header to `<Layout />`

File: src/components/header/notifications.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        color="primary"
+        variant="text"
+        className={classes.seemoreBtn}
+        onClick={props.handleNotifClose}>
+          See More
+      </Button>
+    </>
+  );
+
+  const notifTest = (item) => (
```
Comment (mtalstykh-exadel) [inline src/components/header/notifications.js]: notificationTest isn't so long but easier to read

Comment (fshovchko) [reply src/components/header/notifications.js]: https://github.com/mtalstykh-exadel/team4_frontend/blob/61508cbbb6209a5b4971fa235ad9991b4978da1a/src/components/header/notificationsMenu.js#L29-L34

File: src/pages/profile/profile.jsx (hunk 1 above)
Comment (fshovchko) [reply src/pages/profile/profile.jsx]: https://github.com/mtalstykh-exadel/team4_frontend/blob/61508cbbb6209a5b4971fa235ad9991b4978da1a/src/components/layout/Layout.js#L9-L11

**Review by tensenrey — DISMISSED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Pull request merged**